changelog/update-9704-remove-payouts-rename-overlay

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Remove the overlay that notices the rename from deposits to payouts.
```

Comment:
Nitpick and optional: 

Remove spotlight notice announcing rename of deposit to payout.